.github/pull_request_template.md

```diff
@@ -1,7 +1,9 @@
 Pull request checklist:
- - [ ] Features and bugfixes should be PRed into the `develop` branch, **not** `main`
- - [ ] All text is internationalized
- - [ ] There are no linter errors
- - [ ] New features support all states (loading, error, etc) 
+
+- [ ] Features and bugfixes should be PRed into the `develop` branch, **not** `main`
+- [ ] All text is internationalized
+- [ ] There are no linter errors
+- [ ] New features support all states (loading, error, etc)
+- [ ] Which JIRA ticket(s) and/or github issues does this PR address?
```

Comment:
Based on the phrasing, I'm not sure that this should be a check. Checks are typically yes or no options.